.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
+    "semi": "error",
```

Comment:
thank you very much! Is it possible to add the configuration under https://github.dev/coyotiv/stack? That's what I'd like to follow—including prettier and the following config:

```js
{
  "singleQuote": true,
  "arrowParens": "avoid",
  "printWidth": 120,
  "semi": false,
  "trailingComma": "es5"
}
```

Reply:
Hi @dashersw , this is how I adapted the ESLint rules for these rules. I also updated the eslintrc.js file as json.

```json
   "rules": {
     "quotes": ["error", "single"],
     "arrow-parens": ["error", "always"],
     "max-len": ["error", {
       "code": 120
     }],
     "semi": "error",
     "comma-dangle": ["error"]
   }
```

Reply:
I opened a different branch and added prettier and eslint with the config used in @coyotiv/stack.
I thought about closing this branch as well.